src/legacy/core_plugins/kbn_vislib_vis_types/public/controls/point_series/value_axes.html

```diff
@@ -1,350 +0,0 @@
-
-          <div class="visEditorSidebar__formRow">
-            <label
-              class="visEditorSidebar__formLabel"
-              for="{{ 'valueAxisRotateLabels' + $index }}"
-              i18n-id="kbnVislibVisTypes.controls.pointSeries.valueAxes.rotateLabel"
-              i18n-default-message="Rotate"
-            ></label>
-            <div class="visEditorSidebar__formControl">
-              <select
-                id="{{ 'valueAxisRotateLabels' + $index }}"
-                class="kuiSelect visEditorSidebar__select"
-                ng-model="axis.labels.rotate"
-                ng-options="mode.value as mode.name for mode in rotateOptions"
-              ></select>
-            </div>
-          </div>
-
-          <div class="visEditorSidebar__formRow">
-            <label
-              class="visEditorSidebar__formLabel"
-              for="{{ 'valueAxisTruncateLabels' + $index }}"
-              i18n-id="kbnVislibVisTypes.controls.pointSeries.valueAxes.truncateLabel"
-              i18n-default-message="Truncate"
-            ></label>
-            <div class="visEditorSidebar__formControl">
-              <input
-                id="{{ 'valueAxisTruncateLabels' + $index }}"
-                class="kuiInput visEditorSidebar__input"
-                type="number"
-                ng-model="axis.labels.truncate"
-              >
-            </div>
-          </div>
-
-
-          <!-- Custom Extents -->
-          <h6
-            class="visEditorSidebar__formTitle"
-            i18n-id="kbnVislibVisTypes.controls.pointSeries.valueAxes.customExtentsTitle"
-            i18n-default-message="Custom Extents"
-          ></h6>
-
-          <div class="visEditorSidebar__formRow visEditorSidebar__formRow--expand">
-            <label
-              class="visEditorSidebar__formLabel"
-              for="{ 'defaultYExtents' + $index }"
-              i18n-id="kbnVislibVisTypes.controls.pointSeries.valueAxes.scaleToDataBoundsLabel"
-              i18n-default-message="Scale to Data Bounds"
-            ></label>
-            <div class="visEditorSidebar__formControl">
-              <input class="kuiCheckBox" id="{ 'defaultYExtents' + $index }" type="checkbox" ng-model="axis.scale.defaultYExtents" ng-change="updateBoundsMargin(axis)">
-            </div>
-          </div>
-
-          <div ng-if="axis.scale.defaultYExtents">
-            <div class="visEditorSidebar__formRow">
-              <label
-                class="visEditorSidebar__formLabel"
-                i18n-id="kbnVislibVisTypes.controls.pointSeries.valueAxes.scaleToDataBounds.boundsMargin"
-                i18n-default-message="Bounds Margin"
-              ></label>
-              <div class="visEditorSidebar__formControl">
-                <input
-                  name="yBoundsMargin"
-                  class="kuiInput visEditorSidebar__input"
-                  type="number"
-                  step="0.1"
-                  greater-or-equal-than="0"
```

Comment:
**note**: moved to https://github.com/elastic/kibana/pull/43772/files?file-filters%5B%5D=.tsx&hide-deleted-files=true#diff-0d0929084cab92cbc7b2bba0204b8900R48